### DISCUSSION TITLE
基于Vision OS的空间设计app

### DISCUSSION THREAD

**Discussion by LorieLu** [creatorsand/.github#75] [Ideas]
一句话 IDEA：

AI算法推荐案例库 + AIGC + VISION OS 使空间设计更高效，简易到普通人直接上手设计打造自己的空间。

我的角色：

调研，全流程设计。

其他角色需求（不包含发起人）：

有相关想法经验的志同道合者。

个人简介：

本科上交，英硕回国，现在是不务正业的空间设计师。

**Comment by shallymore** [comment]
超级感兴趣 希望可以一起！
